blog/2024-01-28-reductstore-v1_8_0-released/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "ReductStore v1.8.0 Has Been Released with Data Replication"
+description: ReductStore v1.8.0 introduces data replication between ReductStore instances.
+authors: alexey
+tags: [news]
+slug: news/reductstore-8-released
+date: 2024-01-28
+---
+
+# ReductStore v1.8.0 Has Been Released with Data Replication
```

Comment:
What do you think of "Reductstore V1.8.0 Released: Introducing Data Replication" ?